ftpClientConApp/ftpClientConAppTest/UnitTest1.cs

```diff
@@ -3,24 +3,65 @@
 using System.IO;
 using ftpClientConApp;
 
-namespace ftpClientConAppTest
+namespace ftpClientTests
 {
     [TestClass]
-    public class UnitTest1
+    public class ftpClientConAppTests
     {
-        //private const string Expected = "Hello World!";
+        private const string USERNAME = "anonymous";
+        private const string PASSWORD = "anonymous";
+        private const string SERVERNAME = "ftp://speedtest.tele2.net";

```

Comment:
It's probably best to use localhost to test -- this is something we'll all have access to